style/index.css

```diff
@@ -35,6 +35,22 @@
   box-shadow: inset 0 0px 1px rgba(0, 0, 0, 0.5);
 }
 
+.col-25 {
+  float: left;
+  width: 40%;
```

Comment:
We can probably switch to using `flexbox` at some point to align items in the dialog.

Reply:
Yes, that would be better.